com.github.tenderowl.norka.json

```diff
@@ -87,9 +88,9 @@
                 {
                     "type": "git",
                     "url": "https://github.com/TenderOwl/Norka.git",
-                    "tag": "1.0.1"
+                    "branch": "gtk3"
```

Comment:
Using branches is not a good idea. Ideally it should be tags or releases, but if you need to use it, pin it to a commit using the commit hash. I'm not sure but I think this will be enough to make the lint pass.

Reply:
Yup, just wanna test before merging into the master.